Makefile

```diff
@@ -243,13 +214,12 @@ push-container: build-container
 
 push-tar: build-tar
 	gsutil cp $(TARBALL) $(UPLOAD_PATH)/node-problem-detector/
+	gsutil cp node-problem-detector-$(VERSION)-*.tar.gz* $(UPLOAD_PATH)/node-problem-detector/
```

Comment:
`node-problem-detector-$(VERSION)-*` means that the old `node-problem-detector-$(VERSION).tar.gz` tarball won't be pushed?

Reply:
Good catch! fixed.

Reply:
Actually this was fine. The $(TARBALL) copy above will cover the legacy tarball.

Reply:
ACK